src/Task.java

```diff
@@ -0,0 +1,60 @@
+import utils.DateUtils;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+public class Task extends DateUtils {
```

Comment:
remove extends DateUtils 
read how to use static method in other class

Reply:
complete